bundler.d/assets.rb

```diff
@@ -2,6 +2,7 @@
   gem 'jquery-ui-rails', '~> 6.0'
   gem 'patternfly-sass', '~> 3.59.4'
   gem 'gettext_i18n_rails_js', '~> 1.3.1'
```

Comment:
1.4.0 has been released and I think it's safe to stick to the .y version:
```suggestion
  gem 'gettext_i18n_rails_js', '~> 1.4'
```